uaclient/defaults.py

```diff
@@ -11,6 +11,7 @@
 UAC_ETC_PATH = "/etc/ubuntu-advantage"
 UAC_RUN_PATH = "/run/ubuntu-advantage"
 DEFAULT_DATA_DIR = "/var/lib/ubuntu-advantage"
+NEW_DEFAULT_DATA_DIR = "/var/lib/ubuntu-pro"
```

Comment:
I don't see this directory being removed by a package purge. I would have expected to see it removed in `debian/ubuntu-pro-client.postrm` in the `purge)` case.

Reply:
Good catch. We will fix that.

Reply:
Although purging `ubuntu-advantage-tools` is not really supported... We can do the right thing.
https://github.com/canonical/ubuntu-pro-client/pull/3409 is there